aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -537,11 +537,24 @@ namespace {
            ++a, ++b) {
         auto param_from = *a, param_to = *b;
         AT_ASSERTM(param_from.type() == param_to.type(), "parameter types mismatch");
-        param_to.copy_(param_from.view_as(param_to));
+        if (copy) {
+            param_to.copy_(param_from.view_as(param_to));
+        } else {
+            param_from.resize_as_(param_to);
```

Comment:
Ugh this depends on pretty ugly semantics of `resize_as_`. Generally you shouldn't assume *anything* about the content of a resized tensor, which you do here.

What's even going on in here. Why are you reshaping a *from* tensor to match a *to* tensor?!